pyqtgraph/flowchart/Flowchart.py

```diff
@@ -514,7 +514,7 @@ def loadFile(self, fileName=None, startDir=None):
             return
             ## NOTE: was previously using a real widget for the file dialog's parent, but this caused weird mouse event bugs..
             #fileName = QtGui.QFileDialog.getOpenFileName(None, "Load Flowchart..", startDir, "Flowchart (*.fc)")
-        fileName = unicode(fileName)
+        fileName = str(fileName)
```

Comment:
This fixes python 3 and breaks python 2.
Please use `asUnicode` instead (import from python2_3).